packages/create-emotion/types/tslint.json

```diff
@@ -15,6 +15,7 @@
             "check-typecast",
             "check-type-operator",
             "check-preblock"
-        ]
+        ],
+        "no-null-undefined-union": false
```

Comment:
what's the reason behind disabling this?

Reply:
Interpolations are `undefined | null`. The linting was failing. I can remove one of them if you want?

Reply:
both are needed, so we need to disable this lint rule, like you have done already 👍